transports/webrtc/src/tokio/transport.rs

```diff
@@ -232,7 +246,12 @@ impl ListenStream {
     }
 
     fn poll_if_watcher(&mut self, cx: &mut Context<'_>) -> Poll<<Self as Stream>::Item> {
-        while let Poll::Ready(event) = self.if_watcher.poll_if_event(cx) {
+        let if_watcher = match self.if_watcher.as_mut() {
+            Some(w) => w,
+            None => return Poll::Pending,
+        };
```

Comment:
once we move to Rust 1.65 this can become nicer :-)
```rust
let Some(if_watcher) = self.if_watcher.as_mut() else { return Poll::Pending };
```

Reply:
Yep, looking forward to this, I just didn't want to bump it yet. At least not until we have #3090 where we will start to properly verify our MSRV.